src/manifest.c

```diff
@@ -815,33 +921,41 @@ static int write_manifest_plain(struct manifest *manifest)
 static int write_manifest_tar(struct manifest *manifest)
 {
 	char *conf = config_output_dir();
+	char *files = NULL;
+	char *append = NULL;
 	char *tarcmd = NULL;
 	int ret = -1;
 
 	if (conf == NULL) {
 		assert(0);
 	}
```

Comment:
It looks like there's already a convention... :-)  I'm okay with changing the assert convention separately.